.vib/mediawiki/cypress/cypress/fixtures/users.json

```diff
@@ -0,0 +1,6 @@
+{
+    "newUser": {
+        "username": "User",
+        "password": "password"
```

Comment:
I'd use another non-trivial `password`. In the past, we have faced issues with new versions of applications including assertions to check that the provided password is not trivial, so this way we should be reducing the flakiness of the tests :)